README.rst

```diff
@@ -50,6 +50,16 @@ HistomicsTK can be used in two ways:
   and `examples <https://digitalslidearchive.github.io/HistomicsTK/examples.html>`__
   for more information.
 
+  This can be installed via `pip install histomicstk`. 
```

Comment:
> This can be installed via `pip install histomicstk`.

to 

> This can be installed on Linux via `pip install histomicstk`.

correct?

Reply:
Yes -- we should probably publish a variant without the binaries that would install on non-linux systems when there are appropriate other libraries installed.